packages/tools/client-debugger/client-debugger-view/src/components/Menu.tsx

```diff
@@ -32,6 +32,22 @@ export function MenuSection(props: MenuSectionProps): React.ReactElement {
 	);
 }
 
+/**
+ * Container component for the menu.
+ *
+ * @internal
+ */
+export function ContainerMenuSection(props: MenuSectionProps): React.ReactElement {
```

Comment:
Do we need a separate `ContainerMenuSection` component? The refresh button for the Cotnainers list sits next to the text in the `header` property of the section, so we could update `MenuSection` to take a React component for the `header` instead of a string, and pass in the header text and refresh button there,

Reply:
Yeah, I like Josh's suggestion. The current approach works but feels a bit hacky. It was lucky that `<Menu />` with an empty header worked in this case, but I was not thinking about letting it work without a header when I created it, to be honest 😄.

Reply:
@chentong7 did you have issues implementing Josh's suggestion here? I still think it would be preferrable to splitting the section header from the section contents.

Reply:
Yes. I think it's better to keep the refresh button to header level. Updated here...